modules/cloudwatch_too_many_errors/main.tf

```
@@ -0,0 +1,39 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0.0"
+    }
+  }
+}
+
+
+resource "aws_cloudwatch_log_metric_filter" "error_log_metric_filter" {
+  name           = "ErrorLogMetricFilter"
+  pattern        = "{ $.level = \"ERROR\" }"
```

Hvis du vil være hakket mer konfigurerbar:
```suggestion
  pattern        = "{ $.${var.log_level_field} = \"${var.reported_log_level}\" }"
```

Eller noe sånt. Finn gjerne et bedre navn på variabel-forslaget mitt 😅

---

Ref variabel forslag: Jeg tenker error_tolerance da det er hvor man feil man kan før man mener det burde gå en alarm.

---

Dvs at error_tolerance på 5 betyr at man tåler opp mot 5 feil før det går en alarm